nf_core/lint.py

```diff
@@ -198,7 +198,7 @@ def check_docker(self):
         # Implicitely also checks if empty.
         if 'FROM ' in content:
             self.passed.append((2, "Dockerfile check passed"))
-            self.dockerfile = content.splitlines()
+            self.dockerfile = [line.strip() for line in content.splitlines()]
```

Comment:
Aha, so this is why it wasn't working for me this morning.. 👍

Reply:
yeah, had the same issue in the beginning, thought it would be a good idea using list comprehensions right at the beginning for removing whitespace